ui/src/infra-model/infra-model-main-view.tsx

```diff
@@ -19,7 +21,22 @@ export const InfraModelMainView: React.FC = () => {
                     path="/upload"
                     element={<InfraModelViewContainer viewType={InfraModelViewType.UPLOAD} />}
                 />
-                <Route path="/" element={<InfraModelListContainer changeTimes={changeTimes} />} />
+
+                <Route path="/" element={<TabContainer activeTab={activeTab} />} />
+                {/*
+                <Route
+                    path="/plans"
+                    element={<TabContainer activeTab={InfraModelTabType.PLAN} />}
+                />
+                <Route
+                    path="/waiting-for-approval"
+                    element={<TabContainer activeTab={InfraModelTabType.WAITING} />}
+                />
+                <Route
+                    path="/rejected"
+                    element={<TabContainer activeTab={InfraModelTabType.REJECTED} />}
+                />
+                */}
```

Comment:
Nämä voisi pois kommentoinnin sijasta pistää `EnvRestricted`:in taakse

Reply:
Itse asiassa näitä tuskin on tarvetta edes kätkeä, nyt kun tarkemmin ajattelee